dev/tasks/crossbow.py

```diff
@@ -322,7 +322,11 @@ def create_tree(self, files):
 
     def create_commit(self, files, parents=None, message='',
                       reference_name=None):
-        parents = parents or []
+        if parents is None:
+            # by default use the main branch as the base of the new branch
+            # required to reuse github actions cache across crossbow tasks
```

Comment:
Wow!
Can we use cache with Crossbow and GitHub Actions!? I wanted to use cache to build arm64 deb/rpm.

Reply:
We need to have workflows on the main branch like https://github.com/ursacomputing/crossbow/blob/master/.github/workflows/cache_vcpkg.yml

Preferably we should exercise these kind of builds on a nightly bases and watch out for cache eviction.

Reply:
@kou why did you need GHA cache for ARM64 linux packages? Can we use docker images for caching?

Reply:
> We need to have workflows on the main branch like https://github.com/ursacomputing/crossbow/blob/master/.github/workflows/cache_vcpkg.yml

Ah, I understand.

> why did you need GHA cache for ARM64 linux packages? Can we use docker images for caching?

I want to move off from Travis CI. I think that Buildkite based CI is the best candidate but we don't have enough setup for it yet. So I tried to use GitHub Actions as the second candidate. If we use GitHub Actions for arm64 packages, we need to use QEMU. It's too slow but we'll be able to reduce build time by using ccache. I tried to use ccache but I realized that we can't use cache with Crossbow and GitHub Actions.

Anyway, I want to use cache with Crossbow and GitHub Actions. It can also reduce build time for amd64 packages.

Reply:
The arm machines should be part of the free plan. Is there another reason for moving off of travis?

Reply:
Travis CI's new free plan isn't unlimited.

Crossbow Travis CI builds are disabled again: https://travis-ci.com/github/ursacomputing/crossbow

Reply:
Due to the macOS credits which have been consumed last month.